tinymce-angular-component/src/main/ts/editor/editor.component.ts

```diff
@@ -101,16 +102,15 @@ constructor(
   public ngAfterViewInit() {
     if (isPlatformBrowser(this.platformId)) {
       this.id = this.id || uuid('tiny-angular');
-      this.inline =
-        typeof this.inline !== 'undefined' ? (typeof this.inline === 'boolean' ? this.inline : true) : this.init && this.init.inline;
+      this.inline = this.inline !== undefined ? this.inline !== false : !!(this.init?.inline);
```

Comment:
wow 
!!(this.init?.inline)

is that a maybe init?

Reply:
It's meant to be the same as the original except it coerces it to a boolean rather than occasionally returning `undefined`. 
Essentially `this.init?.inline` means if `this.init` is `undefined` or `null` then return `undefined`, otherwise return the value of `this.init.inline`. Then the double negation causes any `undefined` value to be `false`.

Reply:
Note that according to the types `this.init` can be an object or 'undefined` and then `this.init.inline` can be a boolean or `undefined`.

Reply:
If it wasn't for the slightly odd implication that `this.inline` could be something other than boolean or `undefined` in the original code (and it should resolve to true in that case) the I probably would have simplified it to:
```typescript
this.inline = this.inline ?? this.init?.inline ?? false;
```